sys/net/gnrc/transport_layer/tcp/gnrc_tcp.c

```diff
@@ -391,7 +391,7 @@ ssize_t gnrc_tcp_send(gnrc_tcp_tcb_t *tcb, const void *data, const size_t len,
                 probe_timeout_duration_us = tcb->rto;
             }
             /* Setup probe timeout */
-            _setup_timeout(&probe_timeout, timeout_duration_us, _cb_mbox_put_msg,
+            _setup_timeout(&probe_timeout, probe_timeout_duration_us, _cb_mbox_put_msg,
```

Comment:
Can you briefly explain why this did not affect native?

Reply:
I have not the slightest Idea. I should have happened on native as well. My best guess is that xtimer behaves slightly differently between native and non-native but thats just a guess.